migrations/164_cb_upgrade.js

```diff
@@ -0,0 +1,16 @@
+const { upgradeProxy } = require("@openzeppelin/truffle-upgrades");
```

Comment:
Migration 163 won't work, because game contract needs upgrading before `setTokensPricesAddress` can be called.
We should have this upgrade in migration 163.